server/session/session.go

```diff
@@ -66,8 +67,11 @@ type Session struct {
 	openedWindowID                 atomic.Uint32
 	inTransaction, containerOpened atomic.Bool
 	openedWindow, openedPos        atomic.Value
+	openedContainerID              atomic.Uint32
 	swingingArm                    atomic.Bool
 
+	recipeMapping map[uint32]recipe.Recipe
```

Comment:
I would toss this map out. This just keeps a lot of memory used that isn't used at any other point than at the start to send the recipes.

Reply:
Right now, recipes in the recipe package aren't being stored by their recipe network ID. Should I adjust it to do so, because otherwise I'm unsure on how to do so?

Reply:
You can ignore this, it seemed to be that this map was only used in the `protocolRecipes` method but that was because GitHub collapsed `handler_item_stack_request.go`.